lib/tab_nine_process.py

```diff
@@ -98,6 +95,7 @@ def restart_tabnine_proc(self):
         self.tabnine_proc = TabNineProcess.run_tabnine()
 
     def request(self, req):
+        from json import loads, dumps
```

Comment:
why here as well? Is there an issue with json library?

Reply:
yes, all this PR is related to an issue with Json library, in some cases, another unknown version of it is cashed, and is returned by the import statement. so the solution is to clear it from the cache and reload it